sa/model.go

```diff
@@ -723,10 +724,26 @@ func incidentModelToPB(i incidentModel) sapb.Incident {
 	}
 }
 
-// incidentCertModel represents a row in an 'incident' table.
-type incidentCertModel struct {
+// incidentSerialModel represents a row in an 'incident' table. The fields must
+// match the order of the columns in the 'incident' table schema.
+type incidentSerialModel struct {
 	Serial         string    `db:"serial"`
 	RegistrationID int64     `db:"registrationID"`
 	OrderID        int64     `db:"orderID"`
 	LastNoticeSent time.Time `db:"lastNoticeSent"`
 }
+
+// columns returns a slice of incident table column names. This is used for
+// constructing SQL queries.
+func (i incidentSerialModel) columns() []string {
+	var fields []string
+	t := reflect.TypeOf(i)
+	for i := 0; i < t.NumField(); i++ {
+		f := t.Field(i)
+		dbTag := f.Tag.Get("db")
+		if dbTag != "" {
```

Comment:
Since we're imitating gorp (and using the same tag name), we should use the same semantics: https://github.com/go-gorp/gorp#mapping-structs-to-tables

Specifically, an omitted tag should mean "use the name of the struct field," and `db:"-"` should mean "skip this field."

Reply:
I've taken a different approach that allows us to work with a gorp.TableMap instead of this custom implementation.